### PR TITLE
Make travis cache node_modules to speed up builds

### PR DESCRIPTION
**Pull request by mroderick**
#### Before

2015-10-15 at 16 55

#### After
2015-10-15 at 16 56

### CODE REVIEW

**Comment by mantoni** [comment]
How cool is that! Didn't know this is possible :+1:

File: .travis.yml [hunk 1]
```diff
@@ -7,17 +7,14 @@ node_js:
...
   - npm config set strict-ssl false
   # use a more modern npm than ships with 0.8
   - 'if [ "x$TRAVIS_NODE_VERSION" = "x0.8" ]; then npm install -g npm@1.4; fi'
-  # Workaround for intermittent build failures
-  # install buster separately from `npm install`, as it uses the C compiler, which might cause the
-  #   npm ERR! cb() never called!
-  # errror to appear, causing the builds to fail
-  #
-  # See this discussion: https://github.com/npm/npm/issues/7014
-  - npm install buster@0.7.18
```
Comment (fearphage) [inline .travis.yml]: Will this break when you `rebuild without cache` (or whatever it's called in Travis)? I can't see the button myself.

Comment (mroderick) [inline .travis.yml]: Let's find out!

I've nuked the cache and restarted one of the builds

Comment (mroderick) [inline .travis.yml]: Still works, guess which one I restarted ;-)

2015-10-15 at 21 22

**Pull request merged**